pages/_watch/replay/_replay.vue

```diff
@@ -188,6 +188,8 @@
   import { Player } from '@/store/player';
 
   const Stickers = async () => await import ( '@/components/effects/Stickers' );
+  const g_szHttp = "http";
+  const g_iHttpStringLength = g_szHttp.length;
```

Comment:
creating a variable just to store the length of `http` is unnecessary given it's only used once.

Reply:
I mean, in js we don't have real constants, so, why we should not create it in instance itself instead of creating it each time for constant variable?
Let's imagine that in future you will have some horizontal scroll, that will allow user to load another one replay without reloading page or something like this.

Reply:
`szTempString.length > g_iHttpStringLength` vs `szTempString.length > g_szHttp.length`

IMO, the second way is a little more readable, but I'm not even sure this check is needed since `startsWith()` won't have any issues as long as `szTempString` is a `string`. 

```
szTempString = szTempString.startsWith(g_szHttp) ? szTempString : `https://${szTempString}`;
```

would be my suggestion

Reply:
Yeah, i think startsWith have length checks too. Ok. 

Upd:
Hm, but checked v8 optimizer, anyways, im using firefox, i hope they've optimized it too, haha.